codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/CargoDependency.kt

```diff
@@ -67,10 +67,10 @@ class InlineDependency(
         return extraDependencies
     }
 
-    fun key() = "${module.name}::$name"
+    fun key() = "${module.fullyQualifiedPath()}::$name"
```

Comment:
module.name is old (because modules can only be one level deep). module::fullyQualifiedPath is the new function (module doesn't have a name field anymore)